manifests/anime.my/LavFiltersMegamix/0.75.0-2/anime.my.LavFiltersMegamix.yaml

```
@@ -0,0 +1,24 @@
+PackageIdentifier: anime.my.LavFiltersMegamix
+PackageVersion: 0.75.0-2
+PackageName: LAV Filters Megamix
+Publisher: anime.my
+License: Freeware
+Moniker: lavfiltersmegamix
+Tags:
+- Media Player
+- Codec Pack
+- Multimedia
+ShortDescription: LAV Filters Megamix includes latest directshow video applications like MPC-HC, LAVFilters, ASSFilter, Madvr, PotPlayer, MPDN and some more. It includes wonderful madVR presets too!
+PackageUrl: https://www.videohelp.com/software/LAV-Filters-Megamix
+MinimumOSVersion: 10.0.0.0
+Installers:
+  - Arch: x64
+    InstallerUrl: https://www.videohelp.com/download/Megamix-LAVFilters-0.75.0-2-x64.exe
```

```suggestion
    InstallerUrl: https://www.videohelp.com/download-svVqPtmQP/Megamix-LAVFilters-0.75.0-2-x64.exe
```
The URL you used is not actually a direct link to the binary, there's another redirect for the direct link.

---

The suggested InstallerUrl now redirects to the orignal InstallerUrl.